tests/functional/oauth_choose_redirect.js

```diff
@@ -0,0 +1,54 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+define([
+  'intern',
+  'intern!object',
+  'tests/lib/helpers',
+  'tests/functional/lib/helpers'
+], function (intern, registerSuite, TestHelpers, FunctionalHelpers) {
+  var config = intern.config;
+  var CONTENT_SERVER_ROOT = config.fxaContentRoot;
+
+  var openPage = FunctionalHelpers.openPage;
+  var testElementValueEquals = FunctionalHelpers.testElementValueEquals;
+
+  var oAuthUrl = CONTENT_SERVER_ROOT + 'oauth?client_id=dcdb5ae7add825d2&scope=profile';
```

Comment:
Hard coding the client_id here will break in environments where the client_id is unrecognized. The way around this is to add a `before` function that goes to the OAuth relier, clicks on `signin`, waits for the redirect, and pulls the client_id out of the URL.